src/test/java/com/onaio/steps/utils/Faker.java

```diff
@@ -0,0 +1,62 @@
+package com.onaio.steps.utils;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.nullable;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import android.content.ContentProviderClient;
+import android.content.ContentResolver;
+import android.database.Cursor;
+import android.net.Uri;
+import android.os.RemoteException;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+public class Faker {
+
+    public static void findODKBlankForm(AppCompatActivity activity) throws RemoteException {
```

Comment:
Rename this to `mockQueryInActivityToFindOdkBlankForm`

Reply:
method rename